app/views/likes/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class= "container d-flex flex-column mt-4">
+
+<%= form_with(model: [@post, @like], local: true) do |form| %>
+  <div class="mb-3">
+    <%= form.submit "Save", class: "btn btn-success btn-block" %>
+  </div>
+<% end %>
+
+</div>
```

Comment:
- [ ] A new page that contains a button simply to like a post is not necessary. Kindly implement the like functionality on the Post `show` page without taking the user to this page 👍